custom_components/tahoma/climate.py

```diff
@@ -11,6 +11,8 @@
 ST = "SomfyThermostat"
 DEH = "DimmerExteriorHeating"
 
+TYPE = {AEH: AtlanticElectricalHeater, ST: SomfyThermostat, DEH: DimmerExteriorHeating}
```

Comment:
Any specific reason why this is not noted as:

```python
TYPE = {
	"AtlanticElectricalHeater": AtlanticElectricalHeater, 
	"SomfyThermostat": SomfyThermostat, 
	"DimmerExteriorHeating": DimmerExteriorHeating
}
```

Makes it a bit more readable for me, and easier to extend.

Reply:
because the words were long to type, but fair enough